office/company_events.md

```diff
@@ -1,7 +1,16 @@
 # source{d} Events
 
 ## Get togethers
-Once a month we organise a get together where we either all go to a bar where free drinks will be served from 19:00-24:00 or we go for some group activities like salsa classes, karting or other fun stuff. Every sourcerer and their +1s are invited to join. Get togethers are the third Wednesday of the month. 
+Once a month we organise a get together where we either all go to a bar where free drinks will be served from 19:00-24:00 or we go for some group activities like salsa classes, karting or other fun stuff. That means, every Get together can be different and always depends on the activity.
+Some activities let us be more flexible (beers) while others have to follow a specific schedule (classes). We can generalize:
+  - The event is on the calendar from now to forever, it is the third Wednesday of the month.
+  - The starting time usually is around 7 pm and the duration is around 2-3 hours. 
+  - The goal of this event is to increase the knowledge within the team, there is not a minimum amount of people required,     but if we see there are not a lot of people joining because of the date (not because of the activity) we will try to       change it to be able to put together more people.
+  - We will tell you if the activity is only for sourcerers or your +1s are also invited to join. If the plan is beers,         your +1s are included by default.  
+  - We try to do something nice and cool without being overpriced but we don’t have a specific budget.
```

Comment:
... cannot be provided a wide rank? I understand that having a wide rank does not mean it is needed to use the upper limit, but it would be useful to purpose/discard things (what's the purpose of https://github.com/src-d/guide/issues/98)

Reply:
@eiso @jorgeschnura @mcuadros I don't know if we have it or not. If not, maybe it is a good chance to establish it.